.github/workflows/checkov.yml

```
@@ -0,0 +1,27 @@
+      - name: Run Checkov (All Frameworks)
+        uses: bridgecrew/checkov-action@v3
+        with:
+          directory: ./   
+          quiet: false 
+
+      - name: Run Checkov with SARIF
+        run: checkov -d ./ -o sarif --output-file checkov.sarif
+      - name: Upload SARIF
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+            sarif_file: checkov.sarif
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)